en/docs/develop/error-codes-and-descriptions.md

```diff
@@ -69,7 +69,7 @@ Server.
 | 41001      | Scope name is not specified.                                                                                                                                             |
 | 41002      | Scope display name is not specified.                                                                                                                                     |
 | 41003      | Scope name is not found.                                                                                                                                                 |
-| 41004      | Scope with the name %s already exists in the system. Please use a different scope name.                                                                                  |
+| 41004      | Scope with the name %s already exists in the system. Please use a different scope name.<br> <div class="admonition info"><p class="admonition-title">Info</p> <p>Alternatively if the existing scope is an OIDC scope. The error message will be as follows;<br>"Scope with the name %s already exists as an OIDC scope in the system. Please use a different scope name."</p></div>                                                                                 |
```

Comment:
Let's make this one sentence --> 'Alternatively if the existing scope is an OIDC scope, the error message will be as follows'

Reply:
updated